### ISSUE TITLE
Tooltip misplacement when using Camera2D and zooming

### ISSUE THREAD

**Issue by ShieldHero** [godotengine/godot#11732]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Win7, Godot 2.1.4 stable

**Issue description:**
When using a zoomed Camera2D, it seems that tooltips will be zoomed and their position is wrong, instead of following the cursor it is spawned way off (maybe it uses the transformed mouse position instead), it should use the global mouse position, it doesn't make sense for viewport transformation to be applied on tooltips

**Steps to reproduce:**
Add a Camera2D, change the zoom variable, add a control with a tooltip, when you run it and hover over the control the tool tip will show up in a different position (may be offscreen)

**Comment by DualMatrix** [comment]
Still relevant in master.

**Comment by Calinou** [comment]
Can this be reproduced now that #32559 has been merged? I'm not sure if that pull request fixes it.

**Comment by YeldhamDev** [comment]
@Calinou Nope, and the bug happens not just with zoom, but if the camera is not on the center of the viewport as well.

**Comment by KoBeWi** [comment]
Fixed in 4.0

**Comment by mg979** [comment]
I'm seeing the same just by using a Camera2D and NOT zooming, just moving the camera. The tooltips appear misplaced and sometimes cut and partially off-screen. It seems that tooltips only answer to the absolute position of the mouse position in the scene, not of the viewport. Note that labels aren't affected by this, only their tooltip. Also, I say _viewport_ but I'm not using an actual viewport node (I tried with it too and it doesn't change a thing).

Any chance it can be backported to 3.x if it has been fixed?

![tooltip](https://user-images.githubusercontent.com/26169924/111444297-81adb600-870a-11eb-8ecc-4474ed05ef46.gif)
